itou/scripts/tests.py

```diff
@@ -152,3 +156,93 @@ def test_command_output(self):
         assert pe_approval_1.siae_siret == "123456789"
         assert pe_approval_2.siae_kind == "ETTI"
         assert pe_approval_2.siae_siret == "123456789"
+
+
+class UpdateSuspensionsEndAtTestCase(TransactionTestCase):
+    @freeze_time("2023-02-08")
+    def test_command_output(self):
+        months_of_prolongation = update_suspensions_end_at_mngt_comd.MONTHS_OF_PROLONGATION
+        # One approval per suspension.
+        suspension1 = SuspensionFactory(
+            approval__number="XXXXX0000001",
+            end_at=datetime.datetime.today() + relativedelta(months=Suspension.MAX_DURATION_MONTHS, days=1),
+        )
+        # One approval, 2 suspensions.
+        # The second one should begin when the first one ends.
+        # Only the second one has been updated (but twice).
+        suspension2 = SuspensionFactory(
+            approval__number="XXXXX0000002",
+            start_at=datetime.datetime.today() - relativedelta(months=3),
+            end_at=datetime.datetime.today() - relativedelta(days=1),
+        )
+        suspension3_end_at = datetime.datetime.today() + relativedelta(months=months_of_prolongation * 2)
+        suspension3 = SuspensionFactory(
+            approval=suspension2.approval, start_at=datetime.datetime.today(), end_at=suspension3_end_at
+        )
+
+        # Suspension updated after the first script ran should not be updated.
+        suspension4 = SuspensionFactory(
+            approval__number="XXXXX0000003",
+            start_at=datetime.datetime.today() - relativedelta(days=30),
+            end_at=datetime.datetime.today() - relativedelta(days=1),
+        )
+        suspension5_updated_at = update_suspensions_end_at_mngt_comd.FIRST_SCRIPT_RUNNING_DATE + relativedelta(days=1)
+        suspension5 = SuspensionFactory(
+            approval=suspension4.approval, start_at=datetime.datetime.today(), updated_at=suspension5_updated_at
+        )
+
+        with tempfile.NamedTemporaryFile() as file:
+            suspensions = Suspension.objects.prefetch_related("approval").all()
+            for suspension in suspensions.iterator():
+                log_row = f"{suspension.approval.number} {suspension.start_at} {suspension.end_at}\n"
+                file.write(bytes(log_row, encoding="utf-8"))
```

Comment:
```suggestion
                file.write(log_row.encode())
```
Ou alors tu ouvres le fichier en texte et plus besoin de s’embêter normalement.